### ISSUE TITLE
retinanet-bbox/regression_submodel/pyramid_regression_1/Relu:0_nchwc' Status Message: Input channels C is not equal to kernel channels * group. C: 40 kernel channels: 256 group: 1

### ISSUE THREAD

**Issue by netilovefm1** [dotnet/machinelearning#5413]
### System information

- windows8
- .net core3.1

### Issue

I got a this issue 

but I don't understand what it means ....

how can I change my code ? 

relu is set by keras so I don't understand 
### Source code / logs

retinanet-bbox/regression_submodel/pyramid_regression_1/Relu:0_nchwc' Status Message: Input channels C is not equal to kernel channels * group. C: 40 kernel channels: 256 group: 1

**Comment by frank-dong-ms** [comment]
@netilovefm1 Could you provide a repro of code and possibly training data? What version of ML.NET are you using to cause the error?

**Comment by netilovefm1** [comment]
![image](https://user-images.githubusercontent.com/11716120/94607901-86ef8380-02d7-11eb-8b65-c2bc88264f9c.png)
![image](https://user-images.githubusercontent.com/11716120/94607928-940c7280-02d7-11eb-8042-17769f1d3f58.png)
this layer raise this error

**Comment by frank-dong-ms** [comment]
@netilovefm1 Could you share your repro project so we can take a look?

**Comment by netilovefm1** [comment]
no I just use https://github.com/fizyr/keras-retinanet python source 
I correct some source because h5 model change to ml 


this is regression model 
I didn't correct this part 

def default_regression_model(num_values, num_anchors, pyramid_feature_size=256, regression_feature_size=256, name='regression_submodel'):
    """ Creates the default regression submodel.

    Args
        num_values              : Number of values to regress.
        num_anchors             : Number of anchors to regress for each feature level.
        pyramid_feature_size    : The number of filters to expect from the feature pyramid levels.
        regression_feature_size : The number of filters to use in the layers in the regression submodel.
        name                    : The name of the submodel.

    Returns
        A keras.models.Model that predicts regression values for each anchor.
    """
    # All new conv layers except the final one in the
    # RetinaNet (classification) subnets are initialized
    # with bias b = 0 and a Gaussian weight fill with stddev = 0.01.
    options = {
        'kernel_size'        : 3,
        'strides'            : 1,
        'padding'            : 'same',
        'kernel_initializer' : keras.initializers.RandomNormal(mean=0.0, stddev=0.01, seed=None),
        'bias_initializer'   : 'zeros'
    }

    if keras.backend.image_data_format() == 'channels_first':
        inputs  = keras.layers.Input(shape=(pyramid_feature_size, None, None))
    else:
        inputs  = keras.layers.Input(shape=(None, None, pyramid_feature_size))
    outputs = inputs
    for i in range(4):
        outputs = keras.layers.Conv2D(
            filters=regression_feature_size,
            activation='relu',
            name='pyramid_regression_{}'.format(i),
            **options
        )(outputs)

    outputs = keras.layers.Conv2D(num_anchors * num_values, name='pyramid_regression', **options)(outputs)
    if keras.backend.image_data_format() == 'channels_first':
        outputs = keras.layers.Permute((2, 3, 1), name='pyramid_regression_permute')(outputs)
    outputs = keras.layers.Reshape((-1, num_values), name='pyramid_regression_reshape')(outputs)

    return keras.models.Model(inputs=inputs, outputs=outputs, name=name)

**Comment by frank-dong-ms** [comment]
@netilovefm1 So looks like you are generating a regression model from keras and save the model to onnx format, then you try to load the onnx model in ML.NET and see the exception, is that corrent?
If so, please share your onnx model and whole ML.NET project so we an investigate what is going on here.

**Comment by frank-dong-ms** [comment]
Close this issue since we don't hear back from @netilovefm1 , please feel free to reopen if you can provide more details on this issue.

**Comment by netilovefm1** [comment]
thanks I correct more source it works thanks
